post-processing-code/oncho/testRun.py

```diff
@@ -0,0 +1,38 @@
+import sys
+sys.path.append('../general-functions/')
+from aggregation import aggregatePostProcessedFiles, iuLevelAggregate, countryLevelAggregate, africaLevelAggregate
+from general_post_processing import processSingleFile
+import os
+import pandas as pd
+
+oncho_dir = "../../input-data/oncho/"
+scenarios = os.listdir(oncho_dir)
+scenario_index = 0
+print(f'Total scenarios to process: {len(scenarios)}')
+for oncho_scenario in scenarios:
+    scenario_index += 1
+    for country in os.listdir(os.path.join(oncho_dir, oncho_scenario)):
```

Comment:
💡 It might be good to pull out a function that returns a generator of al the relevant files, so this script becomes a little easier to read and hides the dealing with DS_Store etc away.